azavea/azavea-branding-guide
# Remove HunchLab and assets from Identity > Product section

## Overview
Removal of HunchLab as an Azavea product.

## Testing Instructions

 * `git pull`
 * Go to the Identity > Products page
 * HunchLab should no longer be there.
